tools/sixgill/macros.xml

```diff
@@ -0,0 +1,22 @@
+<macros>
+    <token name="@VERSION@">0.2.4</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="0.2.4">sixgill</requirement>
+            <requirement type="package" version="1.67">biopython</requirement>
+            <requirement type="package" version="0.9.1">pysam</requirement>
```

Comment:
pysam, biopython are not strictly needed here, as this is defined in the conda package